test/e2e/e2e_leaderelection_test.go

```diff
@@ -209,10 +209,10 @@ func startDeschedulerServer(t *testing.T, ctx context.Context, clientSet clients
 		EvictSystemCriticalPods: false,
 		IgnorePvcPods:           false,
 		EvictFailedBarePods:     false,
-		MinPodAge:               &metav1.Duration{Duration: 1 * time.Second},
 	}
-	deschedulerPolicyConfigMapObj, err := deschedulerPolicyConfigMap(podlifetimePolicy(podLifeTimeArgs, evictorArgs))
-	deschedulerPolicyConfigMapObj.Name = fmt.Sprintf("%s-%s", deschedulerPolicyConfigMapObj.Name, testName)
+	deschedulerPolicyConfigMapObj, err := deschedulerPolicyConfigMap(podlifetimePolicy(podLifeTimeArgs, evictorArgs), func(cm *v1.ConfigMap) {
```

Comment:
What's the benefit of abstracting deschedulerPolicyConfigMap and deschedulerDeployment methods if the apply part is used only once?